src/common/module_builder.cc

```diff
@@ -4,55 +4,48 @@
 //
 
 #include "common/module_builder.h"
-#include "status.h"
 
+// c++ standard library includes
 #include <cstdlib>
 #include <iostream>
 
+// loguru includes
+#include "loguru/loguru.hpp"
+
+// llvm mlir includes
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/Func/Extensions/AllExtensions.h"
 #include "mlir/Dialect/Func/IR/FuncOps.h"
 #include "mlir/Dialect/MLProgram/IR/MLProgram.h"
-#include "mlir/IR/MLIRContext.h"
-
-#include "mlir/IR/Attributes.h"              // from @llvm-project
-#include "mlir/IR/Builders.h"                // from @llvm-project
-#include "mlir/IR/BuiltinAttributes.h"       // from @llvm-project
-#include "mlir/IR/BuiltinOps.h"              // from @llvm-project
-#include "mlir/IR/BuiltinTypes.h"            // from @llvm-project
-#include "mlir/IR/MLIRContext.h"             // from @llvm-project
-#include "mlir/IR/OwningOpRef.h"             // from @llvm-project
-#include "mlir/IR/Visitors.h"                // from @llvm-project
-#include "mlir/Parser/Parser.h"              // from @llvm-project
-#include "mlir/Pass/PassManager.h"           // from @llvm-project
-#include "mlir/Support/LLVM.h"               // from @llvm-project
-#include "mlir/Support/LogicalResult.h"      // from @llvm-project
-#include "mlir/Transforms/Passes.h"          // from @llvm-project
-#include "stablehlo/dialect/ChloOps.h"       // from @stablehlo
-#include "stablehlo/dialect/Register.h"      // from @stablehlo
-#include "stablehlo/dialect/Serialization.h" // from @stablehlo
-#include "stablehlo/dialect/StablehloOps.h"  // from @stablehlo
-#include "stablehlo/transforms/Passes.h"     // from @stablehlo
-#include "ttmlir/Dialect/TTIR/Pipelines/TTIRPipelines.h"
-#include "ttmlir/RegisterAll.h"
-
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/DialectRegistry.h"
+#include "mlir/Parser/Parser.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Transforms/Passes.h"
+
+// stablehlo includes
+#include "stablehlo/dialect/Register.h"
+#include "stablehlo/dialect/Version.h"
+#include "stablehlo/transforms/Passes.h"
+
+// tt-mlir includes
 #define TTMLIR_ENABLE_STABLEHLO
```

Comment:
It is on top of tt-mlir includes group because it is affecting only those includes. I agree that it would be more visible if it was somewhere outside, but on the other hand I think it is visible enough by being on top of the group and this way its scope is minimized only to tt-mlir includes group. Let me know if you disagree.